### ISSUE TITLE
Accessibility: use semantic HTML elements

### ISSUE THREAD

**Issue by toebu** [veganaut/veganaut#237]
Many of the newly styled elements are not accessible. The biggest problem is that no button or anchor elements are used. There's many ng-click's on divs or other elements that one can't for example use keyboard navigation.

Also, there are some elements where the clickable area is too small. On the area overview page for example, you can only click on the icons instead of the whole area for getting to the map.

@renestalder: could you review this with the dev team and give inputs how to improve this. At least all clickable elements have to be buttons or anchors, the rest of the accessibility is less of a focus for now.

**Comment by renestalder** [comment]
Concrete "high level" steps:

- Change everything that is related to changing a route to a HTML anchor `<a>` with the correct `href` value.
- Change every other action that is submitting values or triggering UI changes without route changes to a `button[type='button']`, checkbox or radio box, depending on the use case.
- Use appropriate H1, H2, H3, H4,... elements for structuring the website. Apply the correct styling via CSS class selector and **not** via CSS element selector. A H2 can look the same as a H3 in some situations, so the element type doesn't say anything about the look.
- In general: Consult the MDN HTML documentation and use correct semantics.

There are more steps to improve this, but the tasks above should be appropriate for the moment.
Let's make a concrete check on the code when you feel like we have a beta or release candidate.

**Comment by SebuLeugger** [comment]
partially done, partially won´t do
